### DISCUSSION TITLE
Waybar module for show qtile workspaces

### DISCUSSION THREAD

**Discussion by filippoaceto22** [Alexays/Waybar#2610] [General]
Hi, i wanna know if exists a module that show the workspaces in qtile.
If not, i have a function in my config that update a file text somewhere, with groups info. i take this file in a custom module with HTML format and so i m able to show workspaces in that module.

Is possibile to have a real time interval? I'm very noob in waybar.

Thanks!

**Comment by smsteward** [comment]
Since Qtile uses `pywlroots`, you _might_ have some luck using the `wlr/workspaces` module. I don't use Qtile (anymore), so I'm not sure what Wayland protocols they support. But Qtile ships with their own status bar by default...any particular reason you are using Waybar instead of Qtile's built-in bar?

**Reply by filippoaceto22** [reply]
Hi, i use waybar only because i have my config. But i think i can use qtile's bar. I came from Hyprland + Waybar, but i install qtile because, when i use Eclipse idea (guess) , hyprctl loose the session and Hyprland stop showing the bar, because in /tmp/hypr there Is not anymore the socket. I love Hyprland but for work i use Eclipse. Thanks for the reply!
